src/status_im/ui/screens/browser/views.cljs

```diff
@@ -225,7 +225,9 @@
                                                                          (.. ^js % -nativeEvent -data)])
         :on-load                                    #(re-frame/dispatch [:browser/loading-started])
         :on-error                                   #(re-frame/dispatch [:browser/error-occured])
-        :injected-java-script-before-content-loaded (js-res/ethereum-provider (str network-id))}])]
+        :injected-java-script-before-content-loaded (js-res/ethereum-provider (str network-id))
+        ;; https://github.com/status-im/status-mobile/issues/17854
+        :allowsInlineMediaPlayback                  true}])]
```

Comment:
Can we use kebab case here?

Reply:
we can, but I'm not sure it's worth it :)

Reply:
fixed